lib/aika_web/templates/layout/app.html.eex

```diff
@@ -27,6 +27,15 @@
       <div class="alert alert-danger" role="alert"><%= get_flash(@conn, :error) %></div>
     <% end %>
 
+    <div class="api-token">
+      <%= if @user.api_token do %>
+        <p>api token: <%= @user.api_token %></p>
+        <p><%= link "Regenerate Token", to: user_path(@conn, :generate_api_token, @user.id)  %></p>
```

Comment:
As the only difference between these 2 links is the piece of text we could remove the duplication and get the correct text from a view function which can check for the existence of the token